src/containers/Projects/styles.module.scss

```diff
@@ -3,6 +3,12 @@
 .featured {
 }
 
+.heading__main {
+  display: block;
+  margin-bottom: 3rem;
+  composes: h1 from '../../theme/globals.scss';
```

Comment:
Should be `composes: h1 from global;`

Where `global` means _global scope_

Reply:
I'll give it another shot, I tried that way but it didn't worked the styles where not applied.